framework/files/system/oem/01_elemental-rootfs.yaml

```diff
@@ -71,7 +71,7 @@ stages:
     - if: '[ ! -f /oem/userdata ]'
       name: "Pull data from provider"
       datasource:
-        providers: ["cdrom", "gcp", "openstack", "aws", "azure", "hetzner", "packet", "scaleway", "vultr", "digitalocean", "metaldata" ]
+        providers: ["gcp", "openstack", "aws", "azure", "hetzner", "packet", "scaleway", "vultr", "digitalocean", "metaldata" ]
```

Comment:
well, if the edge is elemental's focus, does it make sense to keep data sources at all? The only left here would be just for cloud

Reply:
kind of agreed. config will come from the embedded file always no? not from any datasources.

Reply:
Then let's drop it !
(Well, at least for now ... we might need to resurrect it at a later point :wink:)

Reply:
Good, this will also speed up the boot process, now we were waiting to fail fetching data for all cloud providers 😅 

I am completely cleaning this part then.